app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -56,13 +73,15 @@ function DocumentCaptureWarning({
           />
         }
       >
-        {!!subHeading && subHeading}
-        <UnknownError
-          unknownFieldErrors={unknownFieldErrors}
-          remainingAttempts={remainingAttempts}
-          isFailedDocType={isFailedDocType}
-          hasDismissed={hasDismissed}
-        />
+        <div ref={subheadingRef}>{!!subheading && subheading}</div>
+        <div ref={errorMessageDisplayedRef}>
```

Comment:
This works, another possible way is to  wrap UknownError with forwardRef(looks like it returns <p> element for all scenarios). and create the ref as and avoid create additional <div>

```typescript
const errorMessageDisplayedRef = useRef<HTMLParagraphElement>(null)
```